tests/circuit_graph/test_circuit_graph.py

```diff
@@ -43,32 +42,31 @@ def ops():
     ]
 
 
-@pytest.fixture
-def obs():
+@pytest.fixture(name="obs")
+def obs_fixture():
     """A fixture of observables to go after the queue fixture."""
     return [
         qml.expval(qml.PauliX(wires=0)),
         qml.expval(qml.Hermitian(np.identity(4), wires=[1, 2])),
     ]
 
 
-@pytest.fixture
-def circuit(ops, obs):
-    """A fixture of a circuit generated based on the queue and obs fixtures above."""
-    circuit = CircuitGraph(ops, obs, Wires([0, 1, 2]))
-    return circuit
+@pytest.fixture(name="circuit")
+def circuit_fixture(ops, obs):
+    """A fixture of a circuit generated based on the ops and obs fixtures above."""
+    return CircuitGraph(ops, obs, Wires([0, 1, 2]))
 
 
-@pytest.fixture
-def parameterized_circuit_gaussian(wires):
+@pytest.fixture(name="parameterized_circuit_gaussian")
+def parameterized_circuit_gaussian_name(wires):
```

Comment:
Is this supposed to be fixture like the rest? 
```suggestion
def parameterized_circuit_gaussian_fixture(wires):
```

Reply:
Ah, yes!